Multisig/UI/Assets/Collectibles/CollectibleDetailsView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CollectibleDetailsView.swift
+//  Multisig
+//
+//  Created by Moaaz on 7/14/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct CollectibleDetailsView: View {
+    @ObservedObject
+    var theme: Theme = App.shared.theme
+
+    let viewModel: CollectibleViewModel
+
+    private let imageDimention: CGFloat = 100
+    
+    var body: some View {
+        List {
+            VStack (alignment: .leading, spacing: 7) {
+                TokenImage(width: nil, height: nil, url: viewModel.imageURL, name: "ico-collectible-placeholder")
+                    .frame(minWidth: 0, maxWidth: .infinity, minHeight: 0, maxHeight: .infinity)
+                    .aspectRatio(1.0, contentMode: .fill)
+                    .cornerRadius(10, antialiased: true)
```

Comment:
` antialiased` is `true` by default, so it can be removed, I think this is not the only place where it is used in the code